### PR TITLE
Migrate pull-kubernetes-e2e-gke to json

### PR DESCRIPTION
**Pull request by fejta**
/assign @krzyzacy 

Also update the contexts and retry command to match the job name
Add `--stage-suffix` flag to `kubernetes_e2e.py` now that kubetest supports it.

Migrating these now since they do not automatically run.

### CODE REVIEW

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -305,9 +305,11 @@ def main(args):
         mode.add_k8s(os.path.dirname(k8s), 'kubernetes', 'release')
     if args.stage:
         runner_args.append('--stage=%s' % args.stage)
+    if args.stage_suffix:
+        runner_args.append('--stage-suffix=%s' % args.stage_suffix)
 
     cluster = args.cluster or 'e2e-gce-%s-%s' % (
-        os.environ['NODE_NAME'], os.getenv('EXECUTOR_NUMBER', 0))
+        os.environ['NODE_NAME'], os.getenv('EXECUTOR_NUMBER', 0)))
```
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: ?

File: jobs/pull-kubernetes-e2e-gke.sh [hunk 2]
```diff
@@ -1,84 +0,0 @@
...
-
-export KUBERNETES_PROVIDER="gke"
-export E2E_MIN_STARTUP_PODS="1"
-
-# Flake detection. Individual tests get a second chance to pass.
-export GINKGO_TOLERATE_FLAKES="y"
-export E2E_NAME="e2e-gke-${NODE_NAME}-${EXECUTOR_NUMBER:-0}"
-export GINKGO_PARALLEL="y"
-
-export GINKGO_TEST_ARGS='--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]'
```
Comment (krzyzacy) [inline jobs/pull-kubernetes-e2e-gke.sh]: who handles GINKGO args now?

File: scenarios/kubernetes_e2e.py (hunk 1 above)
Comment (fejta) [inline scenarios/kubernetes_e2e.py]: Done

File: jobs/pull-kubernetes-e2e-gke.sh (hunk 2 above)
Comment (fejta) [inline jobs/pull-kubernetes-e2e-gke.sh]: There is both a `jobs/pull-kubernetes-e2e.env` and `jobs/pull-kubernets-e2e-gke.env`
The `pull-kubernetes-e2e.env` is shared between all the PR e2e jobs.

File: prow/config.yaml [hunk 3]
```diff
@@ -79,9 +79,9 @@ presubmits:
           path: /mnt/disks/ssd0
 
   - name: pull-kubernetes-cross
-    context: Jenkins Cross Build
-    rerun_command: "@k8s-bot cross build this"
-    trigger: "@k8s-bot (cross[ -])?build this"
+    context: pull-kubernetes-cross
+    rerun_command: "@k8s-bot pull-kubernetes-cross test this"
+    trigger: "@k8s-bot pull-kubernetes-cross test this"
```
Comment (krzyzacy) [inline prow/config.yaml]: also update the ones in kubernetes-security below

**Comment by krzyzacy** [comment]
basically LG besides handle k-s repo, I'll need to bounce, feel free to merge after test passes

File: prow/config.yaml (hunk 3 above)
Comment (fejta) [inline prow/config.yaml]: Done

**Pull request merged**